Modules/Core/SpatialObjects/include/itkContourSpatialObject.h

```diff
@@ -158,18 +158,18 @@ class ITK_TEMPLATE_EXPORT ContourSpatialObject
   /** Set the interpolation type */
   itkSetEnumMacro(InterpolationMethod, InterpolationMethodEnum)
 
-    /** Get the interpolation type */
-    itkGetConstMacro(InterpolationMethod, InterpolationMethodEnum)
+  /** Get the interpolation type */
+  itkGetConstMacro(InterpolationMethod, InterpolationMethodEnum)
```

Comment:
```diff
- itkGetConstMacro(InterpolationMethod, InterpolationMethodEnum)
+ itkGetConstMacro(InterpolationMethod, InterpolationMethodEnum);
```

Reply:
Awesome catch Hans :eagle:. Done in da7f9f4. `clang-format` is happy now.